### PR TITLE
Replace unaliased messages

### PR DESCRIPTION
**Pull request by RacerBG**
Ref https://github.com/loot/loot.github.io/issues/76

The following mods were affected by this PR:

AsharaSkyrimCharacterPresetsReplacer.esp
Better_Male_Presets.esp
Kayla_CharPreset.esp
Lagertha_CharPreset.esp
Lydia Face Preset 64bit.esp
SAFO - Valdacil.esp
Immersive Jewelry.esp

The remaining messages seems quite unique. I'm open to suggestions for more unaliased messages if needed.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -8790,12 +8745,7 @@ plugins:
       - Graphics
   - name: 'SAFO - Valdacil.esp'
     msg:
-      - type: warn
-        content:
-          - lang: en
-            text: 'This plugin seems to be outdated and may not be compatible with the latest version of its master files.'
-          - lang: de
-            text: 'Dieses Plugin scheint veraltet zu sein und ist möglicherweise nicht mehr kompatibel mit der aktuellsten Version seiner Master-Dateien.'
+      - <<: &obsolete
```
Comment (sibir-ine) [inline masterlist.yaml]: This message implies that there's an updated version which I don't believe there is. `patchOutdated` would be more appropriate.

```suggestion
      - <<: *patchOutdated
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -4730,33 +4730,15 @@ plugins:
...
-        content:
-          - lang: en
-            text: 'After Character Creation you should deactivate and/or uninstall this.'
-          - lang: de
-            text: 'Nach der Charakter Erstellung sollten Sie es deaktivieren und/oder deinstallieren.'
-          - lang: ja
-            text: 'キャラクターを作成したあと、これを無効化もしくはアンインストールする必要があります。'
-          - lang: pt_PT
-            text: 'Após Criação de Personagem você deve desativar e/ou desinstalar isto.'
+      - <<: &deactivateAfterCharacterCreation
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - <<: *deactivateAfterCharacterCreation
```

File: masterlist.yaml [hunk 3]
```diff
@@ -4730,33 +4730,15 @@ plugins:
...
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - <<: *deactivateAfterCharacterCreation
```

File: masterlist.yaml [hunk 4]
```diff
@@ -4768,16 +4750,7 @@ plugins:
...
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - <<: *deactivateAfterCharacterCreation
```

File: masterlist.yaml [hunk 5]
```diff
@@ -4788,16 +4761,7 @@ plugins:
...
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - <<: *deactivateAfterCharacterCreation
```

File: masterlist.yaml [hunk 6]
```diff
@@ -4809,16 +4773,7 @@ plugins:
...
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - <<: *deactivateAfterCharacterCreation
```

File: masterlist.yaml [hunk 7]
```diff
@@ -15149,12 +15099,7 @@ plugins:
         condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_ImmersiveJewelry_Patch.esp")'
       - <<: *patch3rdParty_KPH_CCOR
         condition: 'active("Complete Crafting Overhaul_Remastered.esp") and not active("Immersive Jewelry_CCOR_Patch.esp")'
-      - type: warn
-        content:
-          - lang: en
-            text: 'Version 1.05 of this mod contains corrupt records. Update to version 1.06a.'
-          - lang: de
-            text: 'Die Version 1.05 dieses Mods enthält korrupte Einträge. Aktualieren Sie zu Version 1.06a.'
+      - <<: &obsolete
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      - <<: *obsolete
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml [hunk 8]
```diff
@@ -15149,12 +15099,7 @@ plugins:
...
       - <<: *patch3rdParty_KPH_CCOR
         condition: 'active("Complete Crafting Overhaul_Remastered.esp") and not active("Immersive Jewelry_CCOR_Patch.esp")'
-      - type: warn
-        content:
-          - lang: en
-            text: 'Version 1.05 of this mod contains corrupt records. Update to version 1.06a.'
-          - lang: de
-            text: 'Die Version 1.05 dieses Mods enthält korrupte Einträge. Aktualieren Sie zu Version 1.06a.'
+      - <<: *obsolete
         condition: 'version("Immersive Jewelry.esp", "1.05", ==)'
```
Comment (sibir-ine) [inline masterlist.yaml]: One last thing I almost forgot, can you please sort this message before the other messages?

Comment (RacerBG) [reply masterlist.yaml]: Done.

File: masterlist.yaml [hunk 9]
```diff
@@ -15145,16 +15095,11 @@ plugins:
...
         condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_ImmersiveJewelry_Patch.esp")'
       - <<: *patch3rdParty_KPH_CCOR
         condition: 'active("Complete Crafting Overhaul_Remastered.esp") and not active("Immersive Jewelry_CCOR_Patch.esp")'
-      - type: warn
-        content:
-          - lang: en
-            text: 'Version 1.05 of this mod contains corrupt records. Update to version 1.06a.'
-          - lang: de
-            text: 'Die Version 1.05 dieses Mods enthält korrupte Einträge. Aktualieren Sie zu Version 1.06a.'
         condition: 'version("Immersive Jewelry.esp", "1.05", ==)'
```
Comment (sibir-ine) [inline masterlist.yaml]: You forgot to take the condition with it :P

Comment (RacerBG) [reply masterlist.yaml]: Still learning... :smile:

**Review by sibir-ine — APPROVED** [review]
Thanks.

**Pull request merged**